dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceConfiguration.java

```diff
@@ -108,6 +108,15 @@
     @NotEmpty
     private String zenodoClientSecret;
 
+    @NotEmpty
```

Comment:
Think build might be dying because these are set as required. 
Either add dummy values to the dropwizard config files used or allow these to be empty.